CMakeLists.txt

```diff
@@ -21,7 +22,7 @@ set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET 10.11)
 
-option(WITH_CXX11ABI "Use the C++11 ABI" OFF)
+option(WITH_CXX11ABI "Use the C++11 ABI" ON)
```

Comment:
Why change this default?

Reply:
It's been long since our development machines and CI have departed from previous C++ implementations using the old ABI. This makes integration with CMake extensions in IDEs like Visual Studio Code much simpler.

Reply:
How does it make this simpler? This option only affects GNU libstdc++

Reply:
Visual Studio Code for Linux

Reply:
> How does it make this simpler? This option only affects GNU libstdc++

VSCode does not use the manual `Makefile`, but instead relies entirely on the CMake configurations from `CMakeLists.txt` and all `*.cmake` files. We run the C++ABI checker script in the Makefile, but not in CMake itself.

Reply:
Ref: https://github.com/mapbox/mapbox-gl-native/blob/master/Makefile#L2

Reply:
Ah sorry, I misread that and assumed you were talking about Visual Studio for Windows